Makefile

```diff
@@ -31,6 +31,7 @@ test:
 	@if test -n "$$CI"; then .tox/py$(PYTHON_MAJOR_MINOR)/bin/coveralls; fi; \
 
 rebuild_updated_recipes: virtualenv
+	. $(ACTIVATE) && \
```

Comment:
It seems that we forgot to activate virtualenv...ahaha...

The two previous PRs should be rebased on top of develop branch once this merged...so we can know for sure that the changes applied there works as expected. The affected PRs are:

  - #2017
  - #2019